.github/workflows/test_on_pr.yml

```diff
@@ -55,6 +55,7 @@ jobs:
         run: |
           wget -q -O prometheus.tgz "https://github.com/prometheus/prometheus/releases/download/v${PROMETHEUS_VERSION}/prometheus-${PROMETHEUS_VERSION}.linux-amd64.tar.gz"
           sudo tar xzf "prometheus.tgz" -C /usr/local/bin --wildcards --wildcards-match-slash --strip-components=1 '*promtool'
+          promtool --version
```

Comment:
Was this leftover from checking, or is it meant to stay?

Reply:
Thought it was a useful thing to leave. Confirms that we've got it installed and working properly.